pages/admin/login.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from 'react';
+import jwt_decode from "jwt-decode";
+const LitJsSdk = require('lit-js-sdk');
+import { Spinner } from '@/common/components/elements/Spinner';
+
+const accessControlConditions = [
+  {
+    method: "balanceOfToken",
+    params: ["6855BL6aaQW9NaJUtbMqyYXnewjWdnSrQxdarKQ54fdD"],
+    chain: "solana",
+    returnValueTest: {
+      key: "$.amount",
+      comparator: ">",
+      value: "0",
+    },
+  },
+];
+
+const resourceId = {
+    baseUrl: process.env.NEXT_PUBLIC_BASE_URL,
+    path: process.env.NEXT_PUBLIC_ADMIN_PATH,
+    orgId: "none",
+    role: "none",
+    extraData: "none"
+}
+
+async function fetchToken() {
+
+    const litNodeClient = new LitJsSdk.LitNodeClient();
+
+
+    const solAuthSig = await LitJsSdk.checkAndSignAuthMessage({ chain: 'solana' });
+
+    const chain = 'solana';
+    const permanent = false;
+
+    const signingArgs = {
+      solRpcConditions: accessControlConditions,
+      chain,
+      authSig: solAuthSig,
+      resourceId
+    }
+
+    await litNodeClient.connect();
```

Comment:
Is this guaranteed not to error? Given that you swallow the error to get the token below, may be worth doing something like that here as well (or just add to the try/catch below)